src/data/balance/pokemon-evolutions.ts

```diff
@@ -99,6 +121,20 @@ export class SpeciesFormEvolution {
     this.item = item || EvolutionItem.NONE;
     this.condition = condition;
     this.wildDelay = wildDelay ?? SpeciesWildEvolutionDelay.NONE;
+    this.tags = this.item === EvolutionItem.NONE
+      ? EvolutionTag.TRIGGER_LEVEL_UP
+      : this.item < EvolutionItem.BLACK_AUGURITE
+        ? EvolutionTag.TRIGGER_USE_ITEM | EvolutionTag.GREAT_ITEM
+        : EvolutionTag.TRIGGER_USE_ITEM | EvolutionTag.ULTRA_ITEM;
+  }
+
+  setTags(tags: EvolutionTag): SpeciesFormEvolution {
```

Comment:
```suggestion
  public setTags(tags: EvolutionTag): SpeciesFormEvolution {
```
Also should have docs.